File: BeeCrowd C++/Iniciante/2483.cpp
#include <iostream>
using namespace std;

string retornar_quantidade_letras(int qtd_a){
  string somatorio_a = "";
  
  for(int i = 0; i < qtd_a; i++){
    somatorio_a += "a";
  }
  return somatorio_a;
}

int main(){
  int qtd_a;
  string resultado_a;

  cin >> qtd_a;

  resultado_a = retornar_quantidade_letras(qtd_a);

  cout << "Feliz nat" << resultado_a << "l!" << endl;
  
  return 0;
}

File: BeeCrowd Python/Iniciante/2483.py
def retornar_quantidade_letras(qtd_a):
  somatorio_a = ""
  for _ in range(qtd_a):
    somatorio_a += "a"

  return somatorio_a

def main():
  qtd_a = int(input())

  resultado_a = retornar_quantidade_letras(qtd_a)

  print(f"Feliz nat{resultado_a}l!")

main()
